.github/workflows/ci.yaml

```diff
@@ -0,0 +1,28 @@
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  run:
+    runs-on: ${{ matrix.operating-system }}
+    strategy:
+      matrix:
+        operating-system: [ubuntu-latest]
+        php-versions: ['7.1', '7.2', '7.3', '7.4']
```

Comment:
We can drop 7.1 as this version of PHP is no longer supported, and App Engine Standard starts at 7.2

Reply:
Dropped 7.1.